pavelib/paver_tests/test_paver_quality.py

```diff
@@ -344,7 +343,6 @@ def test_other_exception(self):
         self._mock_paver_sh.side_effect = [Exception('unrecognized failure!'), 0]
         with self.assertRaises(SystemExit):
             pavelib.quality.run_quality("")
-            self.assertRaises(Exception)
```

Comment:
**Reviewer Note**: I changed these assertions to clearly incorrect assertions and tests still passed.  This code clearly was never run.

Reply:
These are odd. It looks like `assertRaises` is supposed to be used as a context manager, so `self.assertRaises(Exception)` as a statement by itself would *always* be a no-op. I wonder what the intent was, originally!